tests/contrib/pylibmc/test.py

```diff
@@ -371,6 +371,37 @@ def test_patch_unpatch(self):
         assert spans, spans
         assert len(spans) == 1
 
+    def test_client_with_servers_option(self):
+        """
+        A test to make sure we support using `servers`, ie: with pylibmc.Client(servers=[url])
+        """
+        url = "%s:%s" % (cfg["host"], cfg["port"])
+
+        patch()
+        client = pylibmc.Client(servers=[url])
+        # Client should come back as Client(['127.0.0.1:11211'], binary=False)
```

Comment:
Encoding this information in test assertions would clarify its importance. If it's not important enough to do that with, consider removing the comment.

Reply:
I'll delete this comment to dedupe information, I had it while writing the test so I could keep track what to even assert. 👍